### PR TITLE
clarify Glitch instructions

### PR DESCRIPTION
**Pull request by locks**

### CODE REVIEW

File: ember-releases.md [hunk 1]
```diff
@@ -60,10 +60,14 @@ The next release date is not affected by "delays", and should always be calculat
...
+Since generating a new application using ember-cli made Glitch run out of memory,
+the application is cloned from [ember-new-output](https://github.com/ember-cli/ember-new-output).
+
+1. Log in to [https://glitch.com/](https://glitch.com/)
 1. Go to [https://glitch.com/edit/console.html?ember](https://glitch.com/edit/console.html?ember)
-2. Update the local [ember-new-output](https://github.com/ember-cli/ember-new-output) repository
-3. `git checkout` the relevant release version
-4. Include the following snippet for the Glitch button in `app/index.html`
+1. `git fetch`
+1. `git reset --hard v3.22.0` the relevant release version tag
```
Comment (ijlee2) [inline ember-releases.md:69]: Can you turn this line into a sentence? I wasn't sure what you meant by "the relevant release version tag".

File: ember-releases.md [hunk 2]
```diff
@@ -60,10 +60,14 @@ The next release date is not affected by "delays", and should always be calculat
 
 ## Glitch Ember starter
 
+Since generating a new application using ember-cli made Glitch run out of memory,
+the application is cloned from [ember-new-output](https://github.com/ember-cli/ember-new-output).
+
+1. Log in to [https://glitch.com/](https://glitch.com/)
```
Comment (ijlee2) [inline ember-releases.md:66]: I still see the message `"Unauthorized"` when I visit https://glitch.com/edit/console.html?ember . Is there perhaps a permission that needs to be set for my account?

**Review by ijlee2 — APPROVED** [review]
The changes look good to me. I wasn't sure what line 69 meant. Can you write a sentence before merging the pull request?

File: ember-releases.md (hunk 2 above)
Comment (locks) [reply ember-releases.md:66]: To my knowledge adding you to the team should be enough. I was getting a blank screen but after a hard refresh it behaved.
What happens if you try to visit https://glitch.com/edit/#!/emberjs ?

File: ember-releases.md (hunk 1 above)
Comment (locks) [reply ember-releases.md]: Updated!

File: ember-releases.md (hunk 2 above)
Comment (ijlee2) [reply ember-releases.md:66]: https://glitch.com/edit/#!/emberjs led me to a page that looked like an IDE, where I may be to edit the project.

Comment (locks) [reply ember-releases.md:66]: So that means you _should_ have access 🤔

Comment (ijlee2) [reply ember-releases.md:66]: Hmm, might be some login caching issue? If it should work, I think we can merge this PR.

Comment (locks) [reply ember-releases.md:66]: Let's merge and debug out-of-thread!

**Pull request merged**